features/imagebitmap.yml.dist

```diff
@@ -0,0 +1,144 @@
+# Generated from: imagebitmap.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+status:
+  baseline: high
+  baseline_low_date: 2021-09-20
+  baseline_high_date: 2024-03-20
+  support:
+    chrome: "50"
+    chrome_android: "50"
+    edge: "79"
+    firefox: "42"
+    firefox_android: "42"
+    safari: "15"
+    safari_ios: "15"
+compat_features:
+  # baseline: high
+  # baseline_low_date: 2020-01-15
+  # baseline_high_date: 2022-07-15
+  # support:
+  #   chrome: "66"
+  #   chrome_android: "66"
+  #   edge: "79"
+  #   firefox: "46"
+  #   firefox_android: "46"
+  #   safari: "11.1"
+  #   safari_ios: "11.3"
+  - api.ImageBitmapRenderingContext
```

Comment:
I'm skeptical this shipped in Safari 11, before `ImageBitmap` itself in Safari 15. This is probably not correct. Noted in https://github.com/web-platform-dx/web-features/issues/1499